cluster/addons/metadata-proxy/gce/metadata-proxy.yaml

```diff
@@ -38,7 +38,7 @@ spec:
       dnsPolicy: Default
       containers:
       - name: metadata-proxy
-        image: gcr.io/google_containers/metadata-proxy:v0.1.4
```

Comment:
I think this was a bad rebase - metadata proxy now uses a `v` in the tag as of 0.1.4; previously there was no v.

Reply:
good catch